package.json

```diff
@@ -12,7 +12,7 @@
     "build": "run-p compile-external-types compile-v3-contract-types && tsc",
     "clean": "rm -rf dist cdk.out",
     "watch": "tsc -w",
-    "test:unit": "jest test/jest/unit/",
+    "test:unit": "jest test/jest/unit/ && ts-mocha -p tsconfig.cdk.json lib/rpc/**/*.test.ts",
```

Comment:
yea, I think you need to move the tests to `test/jest/unit/`, and then you don't need to modify the `test:unit` script